include/deal.II/fe/fe_poly.templates.h

```diff
@@ -337,20 +337,19 @@ fill_fe_face_values (const typename Triangulation<dim,spacedim>::cell_iterator
 
   if (flags & update_gradients)
     for (unsigned int k=0; k<this->dofs_per_cell; ++k)
-      mapping.transform (make_slice(fe_data.shape_gradients[k], offset, quadrature.size()),
+      mapping.transform (ArrayView<const Tensor<1,dim> >(&fe_data.shape_gradients[k][offset], quadrature.size()),
```

Comment:
There are a number of occurrences of explicit constructions of `ArrayView` here, rather than calls to `make_array_view` as intended. The latter would have been cleaner.

Since I have follow-up patches that already build on this, please ignore this inconsistency when reviewing this patch -- I already have the follow-up patch that cleans this up again.